mm2src/coins/tendermint/tendermint_coin.rs

```diff
@@ -455,8 +463,9 @@ impl From<TendermintCoinRpcError> for BalanceError {
             TendermintCoinRpcError::InvalidResponse(e) => BalanceError::InvalidResponse(e),
             TendermintCoinRpcError::Prost(e) => BalanceError::InvalidResponse(e),
             TendermintCoinRpcError::PerformError(e) => BalanceError::Transport(e),
-            TendermintCoinRpcError::RpcClientError(e) => BalanceError::Transport(e),
-            TendermintCoinRpcError::InternalError(e) => BalanceError::Internal(e),
+            TendermintCoinRpcError::RpcClientError(e) | TendermintCoinRpcError::InternalError(e) => {
+                BalanceError::Internal(e)
+            },
```

Comment:
Q: why rpc clients error now map to internal?
also for `ValidatePaymentError`

note that for validators rpc (`ValidatorsRPCError`) it is mapped to transport error and not internal: https://github.com/KomodoPlatform/komodo-defi-framework/blob/db5cff64dce750f9e2e097869ee32e4350e669a6/mm2src/coins/rpc_command/tendermint/staking.rs#L75

Reply:
I didn't pay attention to this part; it was just a formatting change.